service/awsconfig/v10/resource/lifecycle/create.go

```diff
@@ -121,7 +121,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 				return microerror.Mask(err)
 			}
 
-			n := v1.NamespaceDefault
+			n := customObject.GetNamespace()
```

Comment:
When fetching a `NodeConfig` initially we use the namespace of the `AWSConfig`. Same for creating the `NodeConfig` if it is not there yet.